plugins/org.jboss.tools.openshift.client/.project

```diff
@@ -1,28 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
adding back in. thought it came back in with the last changes you made.  not certain why the gitignore file is still masking it since the patterns in there don't seem to ignore it